*** exception.h.orig	Sun Feb 25 00:17:50 2001
--- exception.h	Sun Feb 25 00:18:29 2001
***************
*** 97,99 ****
--- 97,105 ----
  #define ERROR_GFX_BUSY			1002	// another program uses gfx module
  #define ERROR_GFX_NOTINITED		1003	// videolib not inited yet
  #define ERROR_GFX_TOOMUCH		1004	// your software wants too much, baby
+ 
+ #define ERROR_GGI_INVALID_VISUAL    2000 // No such visual
+ #define ERROR_GGI_OPEN              2001 // Could not open visual
+ #define ERROR_GGI_CONTROL           2002 // Generic control function failure
+ #define ERROR_GGI_OUTPUT            2003 // Generic output function failure
+ #define ERROR_GGI_INPUT             2004 // Generic input function failure
